assets/js/bootstraps/analytics.js

```diff
@@ -6,7 +6,7 @@ import { trackEvent } from '../helpers/metrics';
 function trackDocumentDownloads() {
     const links = document.querySelectorAll('.content-box a[href]');
```

Comment:
I'd be tempted to remove `.content-box` from here too, and look at all links…

Reply:
Works for me!